### DISCUSSION TITLE
Support for thousands-separated string interpolation on floats

### DISCUSSION THREAD

**Discussion by chipnetics** [vlang/v#14639] [Ideas]
Hi, I believe it would be handy to add to the format specifier, a thousands-seperator flag, similar to that of Python and a few other languages.

An example provided by spytheman (in python)....
```
$ cat > x.py
large_number = 4545454.26262666
print(f"Formatted: {large_number:,.2f}")
$ ipython3 x.py 
Formatted: 4,545,454.26
```
Thanks in advance.

**Comment by larpon** [comment]
These can be (in some programming languages) locale dependant. Should V account for this also, in regards to the request?

**Comment by chipnetics** [comment]
Locale is discussed in greater detail on the commuity Discord: https://discord.com/channels/592103645835821068/592294828432424960/953369808295067651

**Comment by larpon** [comment]
Thanks, I missed the discussion before posting 👍
